pillar/params.sls

```diff
@@ -95,3 +95,7 @@ components:
   proxy:
     # Extra arguments to be passed to kube-proxy.
     args: ''
+
+availability_zone:
+  region: 'unknown'
+  zone: 'unknown'
```

Comment:
What about leaving these two parameters empty instead of setting them to `unknown`? Or is `unknown` a key value?